mobile_verifier/src/radio_threshold.rs

```diff
@@ -374,8 +345,7 @@ pub async fn verified_radio_thresholds(
     reward_period: &Range<DateTime<Utc>>,
 ) -> Result<VerifiedRadioThresholds, sqlx::Error> {
     let mut rows = sqlx::query_as::<_, RadioThreshold>(
```

Comment:
We're only grabbing a single field, this query can be simplified

```rs
let gateways = sqlx::query_scalar::<_, PublicKeyBinary>(
        "SELECT hotspot_pubkey
             FROM radio_threshold WHERE threshold_timestamp < $1 and cbsd_id IS NULL",
    )
    .bind(reward_period.end)
    .fetch_all(pool)
    .await?
    .into_iter()
    .collect::<HashSet<_>>();

    Ok(VerifiedRadioThresholds { gateways })
```

And we can get rid of `RadioThreshold`.